src/layout/actions/layout.rs

```diff
@@ -224,6 +254,36 @@ impl LayoutTree {
         self.validate();
     }
 
+    /// Gets the active container and toggles it based on the following rules:
+    /// * If horizontal, make it vertical
+    /// * else, make it horizontal
+    /// This method does *NOT* update the actual views geometry, that needs to be
+    /// done separately by the caller
+    pub fn toggle_active_horizontal(&mut self) {
```

Comment:
`swap_active_container_standard_layout`
`swap_active_container_horizontal_vertical`
`switch_active_container_layout_swap_vertical_with_default_horizontal`
I understand _why_ this needs to exist, but I dunno how else to name it either.